docs/sources/next/javascript-api/k6-timers/_index.md

```diff
@@ -1,16 +1,40 @@
 ---
-title: "k6/timers"
-description: "k6 timers API"
+title: 'k6/timers'
+description: 'k6 timers API'
 weight: 11
 ---
 
 # k6/timers
 
 Implement timers to work with k6's event loop. They mimic the functionality found in browsers and other JavaScript runtimes.
 
-| Function                                       | Description                                                                                    |
-| :------------------------------------------ | :--------------------------------------------------------------------------------------------- |
-| [setTimeout](https://developer.mozilla.org/en-US/docs/Web/API/setTimeout)     | Sets a function to be run after a given timeout.  |
-| [clearTimeout](https://developer.mozilla.org/en-US/docs/Web/API/clearTimeout) | Clears a previously set timeout with `setTimeout`. |
-| [setInterval](https://developer.mozilla.org/en-US/docs/Web/API/setInterval)   | Sets a function to be run on a given interval. |
+| Function                                                                      | Description                                          |
+| :---------------------------------------------------------------------------- | :--------------------------------------------------- |
+| [setTimeout](https://developer.mozilla.org/en-US/docs/Web/API/setTimeout)     | Sets a function to be run after a given timeout.     |
+| [clearTimeout](https://developer.mozilla.org/en-US/docs/Web/API/clearTimeout) | Clears a previously set timeout with `setTimeout`.   |
+| [setInterval](https://developer.mozilla.org/en-US/docs/Web/API/setInterval)   | Sets a function to be run on a given interval.       |
 | [clearInterval](https://developer.mozilla.org/en-US/docs/Web/API/setInterval) | Clears a previously set interval with `setInterval`. |
+
+{{% admonition type="note" %}}
+
+The timing methods are available globally, like in other JavaScript runtimes, and it is unnecessary to import them from the `k6/timers` module.
```

Comment:
```suggestion
The timer methods are available globally, so you can use them in your script without including an import statement.
```

Small edit to make it more direct.

Reply:
This LGTM. 

However instead of `timer methods`, I think `Timing methods` or `Timing functions` is the most common name.

Reply:
I would prefer if we list them by name and say they are all globally available. 

There is a chance in the future we add more to k6/timers and that those *won't* be globally available.

Also goes around figuring out how to name them ... where I think we don't have better way than what the [specification](https://html.spec.whatwg.org/multipage/timers-and-user-prompts.html#dom-settimeout-dev) does:

> The [setTimeout()](https://html.spec.whatwg.org/multipage/timers-and-user-prompts.html#dom-settimeout) and [setInterval()](https://html.spec.whatwg.org/multipage/timers-and-user-prompts.html#dom-setinterval) methods allow authors to schedule timer-based callbacks.

Which doesn't seem great :( either way

Reply:
@ppcano I see more relevant results on Google when I search for "timer methods" over "timing methods". I think it also keeps it consistent with how we're using timers in the rest of the page, like the title "k6 timers API".

@mstoykov I think as long as all the methods currently on this page are globally available, we can keep this admonition as is. In the future, if we include other methods that are not globally available, then we can make the necessary edits here. Another option is to add an additional column to the methods table to specify if they're globally available or not.